### PR TITLE
Added OpenAPI definition

### PR DESCRIPTION
**Pull request by vvalchev**

### CODE REVIEW

**Review by phgeorgiev — APPROVED** [review]

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                value:
+                  exchangeRates:
+                    - EUR:1
+                    - USD:0.987
+                    - GBP:0.878
+                  outputCurrency: GBP
+                  customerVat: "123456789"
+      responses:
+        404:
+          description: There is no client, that matches the provided customer filter.
```
Comment (eroteev) [inline openapi.yaml]:
```suggestion
          description: There are no results that match the provided customer filter.
```

**Review by eroteev — APPROVED** [review]

File: openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+          description: Sum of all documents per customer. If the filter is used, only this customer will be included in the response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CalculateResponse'
+              example:
+                currency: BGN
+                customers:
+                  - name: Vendor 1
+                    vat: 123456789
```
Comment (eroteev) [inline openapi.yaml]:
```suggestion
                    vat: 132457689
```

File: openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                    ballance: 2323.3
+                  - name: Vendor 2
+                    vat: 987654321
```
Comment (eroteev) [inline openapi.yaml]:
```suggestion
                    vat: 987654311
```

**Review by eroteev — CHANGES_REQUESTED** [review]
Looks very good. My suggestion is to remove the usage of the word "client" as "customer" is used throughout the document. It will also be better to avoid using vat numbers which exist in the sample data in the example as well as it might be confusing for the candidates.

**Review by eroteev — APPROVED** [review]

**Pull request merged**